### PR TITLE
update phone number parsing

### PR DESCRIPTION
**Pull request by ros3bud**
This could potentially represent a breaking change, but would also allow flow rules to be more accurate. For example, if phone_1 or phone_2 is blank do or don't do a thing. Now when either is actually blank (work) and (home) will not be present for the field.

### CODE REVIEW

**Comment by cgrayson** [comment]
This is for #40, right?

**Comment by cgrayson** [comment]
Oh, I see, the name of the branch has it, nvm.

File: src/longform.coffee [hunk 1]
```diff
@@ -21,8 +21,8 @@ request = (req) ->
   first_name: lead.Borrower.BorrowerPersonalInformation.Name.FirstName
   last_name: lead.Borrower.BorrowerPersonalInformation.Name.LastName
   email: lead.Borrower.ContactInformation.Email
-  phone_1: "#{lead.Borrower.ContactInformation.HomePhone}h"
-  phone_2: "#{lead.Borrower.ContactInformation.WorkPhone}w"
+  phone_1: if lead.Borrower.ContactInformation.HomePhone then "#{lead.Borrower.ContactInformation.HomePhone}h" else null
+  phone_2: if lead.Borrower.ContactInformation.WorkPhone then "#{lead.Borrower.ContactInformation.WorkPhone}w" else null
```
Comment (cgrayson) [inline src/longform.coffee]: Minor, but I think `"#{lead.Borrower.ContactInformation.HomePhone}h" if lead.Borrower.ContactInformation.HomePhone` has the same effect here.

**Comment by ros3bud** [comment]
Yes -  you got it.

**Comment by cgrayson** [comment]
LGTM. 

As for the breaking change, I'm not 100% sure but I think rules like "…if Phone is blank" won't be affected (i.e., I think `null` is the same as `''` (empty string) in that case). Probably worth verifying that on staging so we're sure.

File: src/longform.coffee (hunk 1 above)
Comment (ros3bud) [inline src/longform.coffee]: I agree.

**Comment by ros3bud** [comment]
Will check things out on staging, for certain.

**Pull request merged**